android/build.gradle

```diff
@@ -31,14 +31,12 @@ subprojects {
         }
 
         dependencies {
-            classpath "com.android.tools.build:gradle:3.4.0-alpha04"
+            classpath "com.android.tools.build:gradle:$gradlePluginVersion"
```

Comment:
I don't know how significant this is, but the `gradle-wrapper.jar` has the SHA256 checksum of the 2.4 version release (https://gradle.org/release-checksums/)

You might want to also upgrade it (`cd android; ./gradlew wrapper --gradle-version 5.4.1`) if you want the latest version, and you can check the checksum hash by using
```
shasum --algorithm=256 gradle/wrapper/gradle-wrapper.jar
```
and then check the hash with the ones available on https://gradle.org/release-checksums/

Reply:
Thanks, I was unaware of this. It added some extra changes to `gradlew` file, but I was able to run, so I guess it is fine 🤷‍♂